const { errorAssert } = require("./error_case_func")

// throw "Test Error"

errorAssert();